src/deploy/functions/tasks.ts

```diff
@@ -125,14 +125,14 @@ export function createFunctionTask(
         if (fn.platform === "gcfv1") {
           await gcf.setIamPolicy({
```

Comment:
This should noop if the policy is `private`

Reply:
Changed the functionality a bit and now private is handled in the `setInvoker` function.